testhelpers/core.go

```diff
@@ -88,18 +89,33 @@ func RequireRandomPeerID(require *require.Assertions) peer.ID {
 	return pid
 }
 
-// TestBlockTimer provides a simple BlockTimer interface implementation.
-type TestBlockTimer struct {
+// TestMessagePoolAPI provides a simple BlockTimer interface implementation.
+type TestMessagePoolAPI struct {
 	Height uint64
 }
 
-// NewTestBlockTimer creates a new TestBlockTimer.
-func NewTestBlockTimer(h uint64) *TestBlockTimer {
-	return &TestBlockTimer{Height: h}
+// NewTestMessagePoolAPI creates a new TestMessagePoolAPI.
+func NewTestMessagePoolAPI(h uint64) *TestMessagePoolAPI {
+	return &TestMessagePoolAPI{Height: h}
+}
+
+type MockMessagePoolValidator struct {
+	Valid bool
+}
+
+func NewMockMessagePoolValidator() *MockMessagePoolValidator {
```

Comment:
I would call this a fake rather than a mock, since it doesn't record or check interactions.